src/Components/TabItem.tsx

```diff
@@ -0,0 +1,24 @@
+import { FC } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+import { Tab } from '../types/Tab';
+
+interface Props {
+  tab: Tab;
+}
+
+export const TabItem: FC<Props> = ({ tab }) => {
+  const { tabId } = useParams();
```

Comment:
You've already used `useParams` in the parent component.
Just provide selectedTabid via props